README.md

```diff
@@ -115,11 +126,20 @@ All configuration options follow. Note that you **RARELY** need to change anythi
   copied from for use with webpack.
 * `optimizeBundle = true`: Whether to use byte code scanner strategy to discover frontend
   components.
-* `pnpmEnable = false`: Instructs to use pnpm for installing npm frontend resources.
+* `pnpmEnable = false` (since 0.7.0): Instructs to use pnpm for installing npm frontend resources.
   pnpm, a.k.a. performant npm, is a better front-end dependency management option.
   With pnpm, packages are cached locally by default and linked (instead of
   downloaded) for every project. This results in reduced disk space usage
   and faster recurring builds when compared to npm.
+* `requireHomeNodeExec = false` (since 0.7.0): Whether vaadin home node executable usage is forced. If it's set to
+  `true` then vaadin home 'node' is checked and installed if it's
+  absent. Then it will be used instead of globally 'node' or locally
```

Comment:
Not blocking: Is the dot after "absent" extra?

Reply:
Hmm, looks good to me :+1: I think I even copy-pasted the text from Flow Java sources somewhere :) I propose to go with this PR and modify the readme in a separate PR if need be.